airbyte-integrations/connectors/destination-bigquery/src/test-integration/java/io/airbyte/integrations/destination/bigquery/BigQueryDestinationTest.java

```diff
@@ -105,10 +106,11 @@
       .withState(new AirbyteStateMessage().withData(Jsons.jsonNode(ImmutableMap.builder().put("checkpoint", "now!").build())));
 
   private static final ConfiguredAirbyteCatalog CATALOG = new ConfiguredAirbyteCatalog().withStreams(Lists.newArrayList(
-      CatalogHelpers.createConfiguredAirbyteStream(USERS_STREAM_NAME, io.airbyte.protocol.models.Field.of("name", JsonSchemaPrimitive.STRING),
+      CatalogHelpers.createConfiguredAirbyteStream(STREAM_NAMESPACE, USERS_STREAM_NAME,
+          io.airbyte.protocol.models.Field.of("name", JsonSchemaPrimitive.STRING),
           io.airbyte.protocol.models.Field
               .of("id", JsonSchemaPrimitive.STRING)),
-      CatalogHelpers.createConfiguredAirbyteStream(TASKS_STREAM_NAME, Field.of("goal", JsonSchemaPrimitive.STRING))));
+      CatalogHelpers.createConfiguredAirbyteStream(STREAM_NAMESPACE, TASKS_STREAM_NAME, Field.of("goal", JsonSchemaPrimitive.STRING))));
```

Comment:
could we create a helper: `CatalogHelpers.streamName(string name)` or `CatalogHelpers.streamName(string namespace, string name)` that returns an `AirbyteStreamName` and pass that object to `createConfiguredAirbyteStream` instead? Code will look better and it is just a matter of changing the constants type now.